src/App.jsx

```diff
@@ -14,70 +16,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const preparedGoods = goods.map((good, index) => ({ good, id: index }
+  ));
```

Comment:
Can we move this variable outside the component to prevent unnecessary re-render